components/migrate/landing/Index.vue

```diff
@@ -2,6 +2,49 @@
   <div class="mt-8">
     <MigrateLandingHeader />
     <hr />
-    <MigrateLandingContent />
+    <div class="flex flex-col md:flex-row gap-4 justify-between">
+      <div>
+        <div
+          v-if="accountId"
+          class="flex flex-col md:flex-row gap-4 items-center">
+          <p class="mr-4">{{ $t('migrate.resultFor') }}</p>
+          <div class="flex">
+            <Avatar :value="accountId" :size="26" class="mr-2" />
+            <nuxt-link
+              class="!text-k-blue"
```

Comment:
```suggestion
              class="text-k-blue"
```

Reply:
Unfortunately, I need `!important` here because the style overridden with global style (`.dark-mode body a`)

![image](https://github.com/kodadot/nft-gallery/assets/734428/7a23e2c6-3e47-4cc2-bb70-5eab430830d2)

Reply:
Interesting, I thought this kind of class name should not be overridden.